boms/cloud-oss-bom/pom.xml

```diff
@@ -44,17 +44,18 @@
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <guava.version>30.1.1-android</guava.version>
-    <google.cloud.java.version>0.158.0</google.cloud.java.version>
-    <google.cloud.core.version>1.95.4</google.cloud.core.version>
+    <guava.version>30.1.1-jre</guava.version>
+    <google.cloud.java.version>0.159.0</google.cloud.java.version>
+    <google.cloud.core.version>2.0.5</google.cloud.core.version>
     <io.grpc.version>1.39.0</io.grpc.version>
     <http.version>1.39.2</http.version>
     <protobuf.version>3.17.3</protobuf.version>
     <!-- We don't use gax-bom because it includes the artifacts with 'testlib' classifier.
         When updating gax.version, update gax.httpjson.version too. -->
-    <gax.version>1.66.0</gax.version>
-    <gax.httpjson.version>0.83.0</gax.httpjson.version>
-    <auth.version>0.26.0</auth.version>
+    <gax.version>2.1.0</gax.version>
+    <gax.httpjson.version>0.86.0</gax.httpjson.version>
+    <auth.version>1.0.0</auth.version>
+    <api-common.version>2.0.1</api-common.version>
```

Comment:
From `<google.api-common.version>2.0.1</google.api-common.version>` in https://search.maven.org/artifact/com.google.cloud/google-cloud-shared-dependencies/2.0.1/pom